.editorconfig

```diff
@@ -3,5 +3,17 @@ root = true
 [*]
 indent_style = tab
 end_of_line = lf
+charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.yml]
+indent_style = space
+indent_size = 2
+
+[{package.json,package-lock.json}]
+indent_style = space
```

Comment:
I recently realized that at some point the npm cli was updated to not clobber your chosen indentation in package.json on every save, so if I'm bothering to specify it explicitly, I'd be a fan of tabs

Reply:
I removed this and updated the package.json file to use tabs.